### PR TITLE
132 meta1 jen3

### PR DESCRIPTION
**Pull request by jenlampton**
Another PR for https://github.com/backdrop/backdrop-issues/issues/132

### CODE REVIEW

File: core/modules/system/system.install [hunk 1]
```diff
@@ -1850,6 +1851,21 @@ function system_update_1011() {
   }
 }
 
+
+/**
+ * Add binary to {date_formats}.format.
+ */
+function system_update_7078() {
```
Comment (quicksketch) [inline core/modules/system/system.install]: This should be excluded and system_last_update_removed() should be updated to require 7078 as the last required update.

**Comment by jenlampton** [comment]
passing tests, assigning to quicksketch for review / commit.

**Comment by jenlampton** [comment]
update removed, retesting

**Comment by jenlampton** [comment]
These tests aren't failing for me locally.  Testning again to see if they were random.

**Comment by quicksketch** [comment]
Everything looks great! Wooo! Merged in.

**Pull request merged**